rskj-core/src/test/java/co/rsk/peg/PegTestUtils.java

```diff
@@ -168,4 +169,23 @@ public static Address createRandomBtcAddress() {
         }
         return keys;
     }
+
+    public static UTXO createUTXO(int nHash, Coin value, Address address) {
+        return new UTXO(
+            createHash(nHash),
+            1,
+            value,
+            0,
+            false,
+            ScriptBuilder.createOutputScript(address));
+    }
+
+    public static List<UTXO> createTestUtxos(int size, Federation federation) {
```

Comment:
```suggestion
    public static List<UTXO> createTestUtxos(int size, Address address) {
```
We only really need an address here, not a federation

Reply:
Resolved